test/healthcheck.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# As this script might be used as docker health check it should exit with either 0/1
```

Comment:
This script seems duplicated in other repo https://github.com/aeternity/aeternity/pull/2635/files

Would it be possible to at least comment in one of the two which one the canonical script is?

Reply:
do you mean https://github.com/aeternity/aeternity/blob/165181713_external_localnet/docker/healthcheck.sh

Reply:
It's different content/script now, this repo holds an extended version and it's used to check this repo only nodes.

The script in the aeternity repo is used to be embedded in the [docker image as `HEALTHCHECK`](https://github.com/aeternity/aeternity/blob/165181713_external_localnet/Dockerfile#L45)